course/models.py

```diff
@@ -59,6 +60,14 @@ class Meta:
         verbose_name = 'Преподователь'
         verbose_name_plural = 'Преподователи'
 
+    def clean(self):
+        image_format = self.image.name.split('.')[-1].lower()
+        if image_format not in settings.ALLOWED_IMAGE_FORMATS:
+            raise ValidationError('Пожалуйста загружите фотографию в формате: jpg, jpeg или png!')
+
+        if self.image.width / 960 != self.image.height / 960:
```

Comment:
There is no need to divide by 960, if they are on both sides. Can be rewritten just as 
`if self.image.width  != self.image.height `

However, if there is some meaning to number 960, then some explanation is needed.